tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchReindexingTests.java

```diff
@@ -150,6 +150,7 @@ public void testReindexerFixesBrokenIndex() throws Exception
 
         // TIDY
         // Restart ElasticsearchConnector.
+        cleanUpIndex();
```

Comment:
This doesn't seem like a good idea. If a test needs there to be no documents in the index then it should clean the index at the start, rather than adding this to the end of every other test.

Reply:
Solved